sdk/core/azure-core/inc/azure/core/azure_assert.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this header be internal? Do we really intend this to be used by the end customer?

Reply:
it's similar to `dll_import_export.hpp` .  We don't have namespaces for includes.